ColorCatalogue.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <assert.h>
+#include "Color.h"
+
+void PrintColorRefrence(int * MajorColorIdx)
+{
+   int lMajorColorIdx;
+   int lMinorColorIdx =0;   
+   lMajorColorIdx = *MajorColorIdx;
+   for( lMinorColorIdx = 0; lMinorColorIdx < numberOfMinorColors; lMinorColorIdx++ )
+   {
+       printf( "%d\t             %s\t            %s\t\n",pairNumber, MajorColorNames[lMajorColorIdx], MinorColorNames[lMinorColorIdx]);
+       pairNumber++;
```

Comment:
pairnumber is set somewhere and used here. This could cause unexpected errors in program if not managed properly. Try to minimise the use of global variables, A well defined function with well named parameters should be used.